### DISCUSSION TITLE
Issue Integrating Dagster with OpenMetadata on Local WSL Environment

### DISCUSSION THREAD

**Discussion by 3angour** [open-metadata/OpenMetadata#18835] [General]
Hello OpenMetadata community,

I'm currently trying to integrate Dagster as a service into OpenMetadata to fetch its metadata. Both OpenMetadata and Dagster are running locally on my WSL Ubuntu environment (on Windows 11). However, I’ve encountered an issue where testing the connection in OpenMetadata consistently fails, and I’m unable to add Dagster successfully.

Here’s a detailed breakdown of the problem:

Setup:
OpenMetadata version: 1.5.10
Dagster version: 1.9.2
Environment: WSL Ubuntu (running on Windows 11)

The Problem:
When I test the Dagster connection in OpenMetadata using either http://localhost:3000/ or http://127.0.0.1:3000/
I always get the following error showed in the image below :
![image](https://github.com/user-attachments/assets/6622a6b6-151f-4b47-8012-d32a988e3090)

However, when I run a curl request directly to the Dagster GraphQL API (http://localhost:3000/graphql) like this : 

curl -X POST http://localhost:3000/graphql \
-H "Content-Type: application/json" \
-d '{
  "query": "query PipelineRuns { repositoriesOrError { __typename ... on RepositoryConnection { nodes { id name location { id name } pipelines { id name description } } } } }"
}'

I can successfully retrieve the pipeline metadata:
Response from Dagster:

{
  "data": {
    "repositoriesOrError": {
      "__typename": "RepositoryConnection",
      "nodes": [
        {
          "id": "423451fde460f7f3e2271e37b7006a84d40efa4a::2efeefa6393af3b161eec7662556feb5758538a0",
          "name": "deploy_docker_repository",
          "location": {
            "id": "example_user_code",
            "name": "example_user_code"
          },
          "pipelines": [
            { "id": "3ba0abeb52911b8035672174f3f13a35c2b2e69e", "name": "__ASSET_JOB", "description": null },
            { "id": "b4013548ab09ca3536a591ea9cafca9a7776011b", "name": "my_job", "description": null },
            { "id": "a29265b92dfdce506d8a4a9e146a97ff0834ee34", "name": "my_step_isolated_job", "description": null }
          ]
        }
      ]
    }
  }
}

1.Has anyone encountered a similar issue when integrating Dagster with OpenMetadata?
2.Could this be related to the WSL setup (networking between OpenMetadata and Dagster)?
3.Are there any additional configurations needed to ensure OpenMetadata can properly fetch metadata from Dagster?

Any guidance or troubleshooting tips would be greatly appreciated.

Thank you!

**Comment by Prajwal214** [comment]
@3angour Did you deploy OM on docker or locally?
